backend/src/config/components/urlBuilder.js

```diff
@@ -0,0 +1,36 @@
+import env from '../../constants/env.js';
+
+const getValueByEnviroment = () => {
+  const enviroment = env.NODE_ENV;
+  const config = {};
+
+  config.username = env[`DB_USER_${enviroment}`];
+  config.password = env[`DB_PASS_${enviroment}`];
+  config.host = env[`DB_HOST_${enviroment}`];
+  config.port = env[`DB_PORT_${enviroment}`];
+  config.name = env[`DB_NAME_${enviroment}`];
+
+  return config;
+};
+
+const urlBuilder = () => {
```

Comment:
It is not a proper naming - in this case, it is just a function (should be named like getConnectionUrl)

If you want to use urlWrapper name treat module file as a class (so everything should be wrapped in UrlBuilder